### ISSUE TITLE
There's bugs with rage options.. I think.

### ISSUE THREAD

**Issue by jeong760** [kanhavishva/KeyHunt-Cuda#32]
Dear,

If I set the key range then it should start from that key range but it always starts from the beginning.
Could you tell me how I start with designated key space such as 64bit?

Best regards,
jeong760

**Comment by Noname400** [comment]
use the parameter (-r)

**Comment by jeong760** [comment]
however, -r mode is randomly changes the keyspace after a settled number of keys. 
so just wonder why I use a -r option to start from 64bit to 160bit private key searching?

**Comment by Noname400** [comment]
as the developer explained. the parameter changes keys only within the range.
how did you determine that it is out of range?

**Comment by jeong760** [comment]
Dear,
​
I will make a test but I am not meaning of randomly chnaged keyspace by the way. I asked that why always start from the beginning when I restart the program even if there set start from .... 
​
-----Original Message-----

**Comment by Noname400** [comment]
this number indicates the number of keys tested. and nothing more.
in the picture you went through 7 billion keys, that's 43 bits

**Comment by jeong760** [comment]
Yes... but I settled to start from 64bit puzzled key space so...
​
-----Original Message-----

**Comment by kanhavishva** [comment]
![image](https://user-images.githubusercontent.com/18043565/130437794-7bd112e1-3a6d-4e28-8895-d3e6681ec1be.png)

this is number of total scanned keys and bit value also indicates total scanned keys bit length, it's not related to given range.

**Comment by Mehdi256** [comment]
@kanhavishva 
I asked for the keys to be displayed some time ago, but the manufacturer says this is not possible  :(

**Comment by jeong760** [comment]
Dear,
​
Alright, thanks anyway.
​
Have a pleasant days with some lucks 
​
-----Original Message-----
